### PR TITLE
Add support for NVMe volumes to mounting script

### PR DESCRIPTION
**Pull request by brianherrera**
This PR updates the inc_build_util script to support build nodes that utilize NVMe-based storage. Required when we migrate all our linux nodes from C4 to to C5. Nodes that still use the current device setup will continue operating normally. 

Tested in our fork:
- Running linux build on c5.9xlarge prior to update: build failed
- Running linux build on c5.9xlarge after update: build succeeds
- Running linux build on c4.8xlarge after update: build succeeds

### CODE REVIEW

**Review by AMZN-alexpete — APPROVED** [review]
In the future will we bypass the code that attaches/detaches the xvdf volumes if an nvme is available?

**Review by amznestebanpapp — APPROVED** [review]

File: scripts/build/bootstrap/incremental_build_util.py
Comment (amznestebanpapp) [inline scripts/build/bootstrap/incremental_build_util.py:330]: doesnt line 335 have to be updated as well, I am also seeing mention to xvdf other times
`delete_branch_ebs.py` script is also mentioning that device

**Review by amznestebanpapp — CHANGES_REQUESTED** [review]

File: scripts/build/bootstrap/incremental_build_util.py
Comment (brianherrera) [reply scripts/build/bootstrap/incremental_build_util.py:330]: It doesn't need to be updated there since that's just the device name provided to EBS. It will then be converted to the required device name in Linux or drive letter in Windows. 

I'll take a look at delete_branch_ebs.py and our other maintenance scripts as well.

Comment (amznestebanpapp) [reply scripts/build/bootstrap/incremental_build_util.py:330]: I will then mark this as approved, leave it up to you to figure if it is needed to update those other scripts at the same time

**Review by amznestebanpapp — APPROVED** [review]

**Comment by brianherrera** [comment]
> In the future will we bypass the code that attaches/detaches the xvdf volumes if an nvme is available?

When all our platforms use the nvme storage we can remove it. Leaving it there for now while we support both storage platforms.

File: scripts/build/bootstrap/incremental_build_util.py
Comment (brianherrera) [reply scripts/build/bootstrap/incremental_build_util.py:330]: Sounds good. I'm reviewing them now, at a quick glance doesn't look like it will affect the cleanup scripts since this update only affects how it mounts the local device. But I'll double check before I merge it in.

**Pull request merged**
